packages/world/src/systems/BuildSystem.sol

```diff
@@ -147,52 +168,53 @@ contract BuildSystem is System {
       SeedGrowth._setFullyGrownAt(base, uint128(block.timestamp) + ctx.buildType.getTimeToGrow());
     } else if (ctx.buildType.hasExtraDrops()) {
       DisabledExtraDrops._set(base, true);
+    } else if (ctx.buildType == ObjectTypes.ForceField) {
+      (EntityId existing,) = ForceFieldUtils.getForceField(ctx.coord);
```

Comment:
Doesn't this check need to run in all coords not just base, since the force field tower could take up more than 1 position? or we're assuming base is where its at, similar to beds?

Reply:
yes I think we should always keep it at 1, because even if it used more voxels we should only setup a single fragment as the "main" fragment and expand from there. I think we should keep forcefields as a single block forever, otherwise it becomes trickier if the forcefield is in multiple fragments.